recipes/journey_2/module_1/probe.yaml

```diff
@@ -0,0 +1,623 @@
+tasks:
+  - name: "Question #1"
+    context: "How to provide context to GitHub Copilot for better code suggestions."
```

Comment:
Add more context about the quiz like the learner is doing a quiz, don't tell them a direct answer, etc. Context should also include what the journey is roughly about